doc/source/conf.py

```diff
@@ -135,6 +135,7 @@ def _stop_fluent_container(gallery_conf, fname):
             .split()
         )
         for container_name in container_names:
+            print(f"Container still running for script {fname}")
```

Comment:
```suggestion
            print(f"Container still running for script {fname}".)
```
Message strings should have concluding punctuation.